app/views/employees/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="container">
+
+  <div class="row">
+    <div class="col-sm-12">
+      <!-- Alert bar -->
+      <div class="alert alert-success alert-dismissable present" id="present_alert" style="display: none;">
+         
+        <button type="button" class="close" data-dismiss="alert" aria-hidden="true">
+          ×
+        </button>
+        <h4>
+          Success!
```

Comment:
why always "success"?

Reply:
it only appears when attendance is marked, else an error alert is shown through js